packages/cra-template-imodeljs/README.md

```diff
@@ -0,0 +1,20 @@
+# cra-template-typescript
+
+This is the official TypeScript template for [Create React App](https://github.com/facebook/create-react-app).
+
+To use this template, add `--template typescript` when creating a new app.
+
+For example:
+
+```sh
+npx create-react-app my-app --template typescript
```

Comment:
should probably mention you can set the scripts version so it uses the bentley fork one